ansible/roles/munin_node/handlers/main.yml

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: MIT
+- name: 'systemd restart munin-node'
+  systemd:
+    name: munin-node
+    state: restarted
+- name: 'systemd reload apache2'
+  systemd:
+    name: apache2
+    state: reloaded
+- name: 'systemd restart apache2'
+  systemd:
+    name: apache2
+    state: restarted
+- name: 'systemd daemon-reload'
+  systemd:
+    daemon-reload: yes
```

Comment:
Do we need this 3 times? And this is the only one that uses ```daemon-reload``` instead of ```daemon_reload```.

Reply:
I added a special "handlers" role where all handlers for Apache and Systemd are available because they are used most.